src/Nox.Types/Types/Month/Month.cs

```diff
@@ -35,4 +37,25 @@ public override string ToString()
     {
         return Value.ToString("00");
     }
```

Comment:
Can we have overloading of "From" method that accepts "int" value?

Reply:
Required changes were applied